autovalet_navigation/config/sim/costmap_common_params.yaml

```diff
@@ -15,11 +15,25 @@ inflation: # Name of the inflation layer plugin in global costmap params
   cost_scaling_factor: 1.0
 
 obstacle_layer:
+    observation_sources: lane
     max_obstacle_height: 1.5
-    min_obstacle_height: 0.2
-
-observation_sources: realsense velodyne lane
+    min_obstacle_height: -0.2
```

Comment:
why is min obstacle height negative(-0.2)?

Reply:
you're right. It should be reverted to some positive values. It was changed to do some sanity check. good catch!